etl/steps/data/garden/survey/2024-04-01/dietary_choices_uk.meta.yml

```diff
@@ -0,0 +1,94 @@
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      attribution_short: YouGov
+      topic_tags:
+        - Diet Compositions
+        - Animal Welfare
+        - Food Supply
+    display:  &common-display
+      numDecimalPlaces: 0
+
+dataset:
+  update_period_days: 365
```

Comment:
not sure if you are only planning to update this once a year only anway, but the update frequency of these trackers is usually 6 months

Reply:
I think for us it's enough to update this once a year.